frontend/src/features/properties/components/forms/subforms/AddressForm.tsx

```diff
@@ -74,9 +74,8 @@ const AddressForm = <T extends any>(props: AddressProps & FormikProps<T>) => {
   return (
     <>
       <Row className="field-row">
-        <Label required>Street Address</Label>
+        <Label required={!props.verticalLine}>Street Address</Label>
```

Comment:
The vertical line is a prop explicitly used on the review page to separate the disabled inputs and the label - this may need to change as it is confusing to read

Reply:
Will think of a better way to do this in the morning tomorrow